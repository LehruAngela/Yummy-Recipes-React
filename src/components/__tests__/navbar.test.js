import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import Navbar from '../navbar';

describe('Navbar', () => {
    const wrapper = shallow(<Navbar/>);
    const preventDefault = jest.fn();

    it('renders properly', () => {
        expect(shallowToJson(wrapper)).toMatchSnapshot();
        }); 
    

})