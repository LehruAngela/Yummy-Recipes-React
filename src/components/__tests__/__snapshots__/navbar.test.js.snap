// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders properly 1`] = `
<nav
  class="navbar navbar-expand-lg navbar-light bg-light"
>
  <div
    className="black"
  >
    <a
      class="navbar-brand"
    >
      Yummy Recipes
    </a>
  </div>
  <button
    aria-controls="navbarSupportedContent"
    aria-expanded="false"
    aria-label="Toggle navigation"
    class="navbar-toggler"
    data-target="#navbarSupportedContent"
    data-toggle="collapse"
    type="button"
  >
    <span
      class="navbar-toggler-icon"
    />
  </button>
  <div
    class="collapse navbar-collapse"
    id="navbarSupportedContent"
  >
    <ul
      class="navbar-nav mr-auto"
    >
      <li
        class="nav-item active"
      >
        <a
          class="nav-link"
          className="btn btn-light"
          href="/categories"
        >
          Categories
          <span
            class="sr-only"
          >
            (current)
          </span>
        </a>
      </li>
    </ul>
    <Profile />
    <withRouter(Logout) />
  </div>
</nav>
`;
